tests/roles/run_tests/templates/cluster-template-controlplane-kubeadm-config-centos.yaml

```
@@ -1,8 +1,10 @@
 # CentOS specific controlplane kubeadm config
 preKubeadmCommands:
   - systemctl restart NetworkManager.service
-  - nmcli connection load /etc/sysconfig/network-scripts/ifcfg-eth0
-  - nmcli connection up filename /etc/sysconfig/network-scripts/ifcfg-eth0
+  - nmcli con add load /etc/NetworkManager/system-connections/eth0.nmconnection
+  - nmcli connection up filename /etc/NetworkManager/system-connections/eth0.nmconnection
+  - nmcli con add load /etc/NetworkManager/system-connections/{{ IRONIC_ENDPOINT_BRIDGE }}.nmconnection
+  - nmcli connection up filename /etc/NetworkManager/system-connections/{{ IRONIC_ENDPOINT_BRIDGE }}.nmconnection
```

This looks weird to me, like there are multiple commands on the same line or something. For example I think it should be `nmcli con load` (without the `add`).

```suggestion
  - nmcli connection load /etc/NetworkManager/system-connections/eth0.nmconnection
  - nmcli connection up eth0
  - nmcli connection load /etc/NetworkManager/system-connections/{{ IRONIC_ENDPOINT_BRIDGE }}.nmconnection
  - nmcli connection up {{ IRONIC_ENDPOINT_BRIDGE }}
```

---

changed it